### PR TITLE
Add checks for epoch start config when init bootstrap

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Added checks for epoch start configuration when starting the bootstrap component. Otherwise, this could occur in late errors.

### CODE REVIEW

File: epochStart/bootstrap/epochStartMetaBlockProcessor.go [hunk 1]
```diff
@@ -63,10 +65,10 @@ func NewEpochStartMetaBlockProcessor(
 	if !(consensusPercentage > 0 && consensusPercentage <= 100) {
 		return nil, epochStart.ErrInvalidConsensusThreshold
 	}
-	if minNumConnectedPeers < 1 {
+	if minNumConnectedPeers < minNumConnectedPeers {
```
Comment (SebastianMarian) [inline epochStart/bootstrap/epochStartMetaBlockProcessor.go:68]: Could you rename with caps M the constant? They have the same name in left and right side of this compare

File: epochStart/bootstrap/epochStartMetaBlockProcessor.go [hunk 2]
```diff
@@ -63,10 +65,10 @@ func NewEpochStartMetaBlockProcessor(
...
 		return nil, epochStart.ErrNotEnoughNumConnectedPeers
 	}
-	if minNumOfPeersToConsiderBlockValid < 1 {
+	if minNumOfPeersToConsiderBlockValid < minNumOfPeersToConsiderBlockValid {
```
Comment (SebastianMarian) [inline epochStart/bootstrap/epochStartMetaBlockProcessor.go:71]: Could you rename with caps M the constant? They have the same name in left and right side of this compare

File: epochStart/bootstrap/epochStartMetaBlockProcessor.go (hunk 1 above)
Comment (bogdan-rosianu) [reply epochStart/bootstrap/epochStartMetaBlockProcessor.go]: you're right. good find. renamed the received parameter and kept the constant name the same

File: epochStart/bootstrap/epochStartMetaBlockProcessor.go (hunk 2 above)
Comment (bogdan-rosianu) [reply epochStart/bootstrap/epochStartMetaBlockProcessor.go]: renamed the parameter

**Pull request merged**